### PR TITLE
adding laxpy and pyfor

### PR DESCRIPTION
**Pull request by brycefrank**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/laxpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+requirements:
+  host:
+    - geopandas
+    - laspy
+    - lastools
+    - numba
+    - numpy
+    - shapely
+    - python >=3.6
+    - pip
```
Comment (marcelotrevisani) [inline recipes/laxpy/meta.yaml]: In ``host`` section I think you just need ``laspy``, ``python`` and ``pip``

Comment (brycefrank) [reply recipes/laxpy/meta.yaml]: I was hoping to get some advice there. I will make the modifications soon.

File: .gitignore [hunk 2]
```diff
@@ -1,14 +0,0 @@
-# macOS folder metadata
-.DS_Store
-
-# User builds
-build_artifacts
-
-# Compiled Python code
-__pycache__
-*.pyc
```
Comment (marcelotrevisani) [inline .gitignore]: please, revert this file

File: .vs/ProjectSettings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```
Comment (marcelotrevisani) [inline .vs/ProjectSettings.json]: please, exclude this file

File: .vs/VSWorkspaceState.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "ExpandedNodes": [
+    "",
```
Comment (marcelotrevisani) [inline .vs/VSWorkspaceState.json]: please, remove this folder ``.vs``

**Pull request merged**